### ISSUE TITLE
example model MD5 sum does not match model_md5

### ISSUE THREAD

**Issue by copernican** [metrumresearchgroup/rbabylon#140]
As discussed in #30, the value of `model_md5` in `tests/testthat/model-examples/1/bbi_config.json` is not the MD5 sum of `tests/testthat/model-examples/1.ctl`.

```sh
➜  rbabylon md5sum tests/testthat/model-examples/1.ctl
6b930119c4224ba077091b47959b0604  tests/testthat/model-examples/1.ctl
➜  rbabylon grep model_md5 tests/testthat/model-examples/1/bbi_config.json
    "model_md5": "731923458236cc008c3adafa2f0877a7",
```

The MD5 sum stored in `bbi_config.json` corresponds to `tests/testthat/model-examples/1/1.ctl`.

```sh
➜  rbabylon md5sum tests/testthat/model-examples/1/1.ctl
731923458236cc008c3adafa2f0877a7  tests/testthat/model-examples/1/1.ctl
```

This may be related to a since-resolved issue in babylon (possibly metrumresearchgroup/babylon#168), and may be fixed by re-running the model with a recent version.

**Comment by copernican** [comment]
Re-running the model is complicated by the fact that the version of babylon originally used to run the test model is incompatible with the latest release of `rbabylon`.

```r
library(rbabylon)
model_dir <- "/home/seanw/git/rbabylon/tests/testthat/model-examples/"
file.copy(file.path(model_dir, "1/babylon.yaml"), model_dir)
#> [1] TRUE
options(rbabylon.bbi_exe_path = "/data/apps2/bbi")
set_model_directory(model_dir)
#> options('rbabylon.model_directory') set to /data/home/seanw/git/rbabylon/tests/testthat/model-examples
submit_model(1, .bbi_args = list(overwrite = TRUE), .mode = "local")
#> Error: The executable at `/data/apps2/bbi` is version 2.1.2 but the minimum supported version of babylon is 2.2.0
#> Call `use_bbi('/data/apps2')` to update to the most recent release.
```

But if I attempt to run with the latest release of babylon (currently, `2.2.0-beta2`), running `bbi_init()` prior to `submit_model()` results in material differences to `babylon.yaml`. To see the differences, copy `babylon.yaml` as above, then run

```r
bbi_init(model_dir, "/opt/NONMEM", "nm74gf")
```

and

```sh
diff babylon.yaml 1/babylon.yaml
```

If we then run `submit_model()`, we see numerous differences to the contents of the output directory. #145 contains the results of running the model. We should inspect the results and determine if the differences are material.

**Comment by copernican** [comment]
closed by 0b666af
